wbooks/src/app/index.tsx

```diff
@@ -1,100 +1,22 @@
-/* eslint-disable no-use-before-define */
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- *
- * Generated with the TypeScript template
- * https://github.com/react-native-community/react-native-template-typescript
- *
- * @format
- */
+import React, { FC } from 'react';
+import { SafeAreaView, ScrollView, View, StatusBar } from 'react-native';
+import LibaryScreen from '@screens/LibraryScreen/LibraryScreen';
 
-import React from 'react';
-import { SafeAreaView, StyleSheet, ScrollView, View, Text, StatusBar } from 'react-native';
-import {
-  Header,
-  LearnMoreLinks,
-  Colors,
-  DebugInstructions,
-  ReloadInstructions
-} from 'react-native/Libraries/NewAppScreen';
+import styles from './styles';
 
-const App = () => {
+const App: FC = () => {
   return (
     <>
-      <StatusBar barStyle="dark-content" />
-      <SafeAreaView>
+      <StatusBar backgroundColor="transparent" translucent barStyle="dark-content" />
+      <SafeAreaView style={styles.container}>
         <ScrollView contentInsetAdjustmentBehavior="automatic" style={styles.scrollView}>
```

Comment:
same, is this really nedeed? If it is, you should add it to the Library screen, not here